src/app/components/champion/ChampionView.tsx

```diff
@@ -67,53 +89,108 @@ export const ChampionView: FC<ChampionProps> = ({champion, callback, championsLi
 
             <ListItem key='IDCard'>
                 <ListItemAvatar>
-                    <Avatar alt={champion.name} variant='square'
-                            src={"http://ddragon.leagueoflegends.com/cdn/11.6.1/img/champion/"+champion.name+".png"}/>
+                    <Avatar className={classes.avatarDisplay} alt={champion.name} variant='square'
+                            src={"https://ddragon.leagueoflegends.com/cdn/11.6.1/img/champion/" + champion.name + ".png"}/>
```

Comment:
Doesnt it bother you that you have exactly the same compute 20 lines before ?

Reply:
I will deal with it in my DataDragon API ticket(s).